js/localization.d.ts

```diff
@@ -117,4 +117,11 @@ type ExternalFormat = any;
  * @namespace DevExpress.ui
  * @public
  */
-export type Format = FormatObject | PredefinedFormat | string | ((value: number | Date) => string) | ExternalFormat;
+export type Format =
+  FormatObject |
+  PredefinedFormat |
+  string |
+  ((value: number | Date) => string) |
+  ((value: Date) => string) |
+  ((value: number) => string) |
+  ExternalFormat;
```

Comment:
Wait. You just said, that ExternalFormat is never. Never doesn't do anything in unions - it's like empty set. I suggest to just don't write it in the union.

Reply:
`ExternalFormat` is a stub for external format types that we're going to add in future.